src/ui/font.rs

```diff
@@ -47,8 +47,8 @@ impl Font {
                 match ch {
                     'h' => {
                         let rest = chars.collect::<String>();
-                        let h = rest.parse::<usize>().or(Err(()))?;
-                        if h == 0 {
+                        let h = rest.parse::<f32>().or(Err(()))?;
+                        if h == 0.0 || h < 0.0 {
```

Comment:
`h <= 0.0` is enough.

Reply:
@vhakulinen Thank you! Should be fixed now.